server/channels/api4/channel.go

```diff
@@ -1460,9 +1459,8 @@ func getChannelMember(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	ctx := c.AppContext
-	ctx.SetContext(app.WithMaster(ctx.Context()))
-	member, err := c.App.GetChannelMember(ctx, c.Params.ChannelId, c.Params.UserId)
+	c.AppContext = app.RequestContextWithMaster(c.AppContext)
+	member, err := c.App.GetChannelMember(c.AppContext, c.Params.ChannelId, c.Params.UserId)
```

Comment:
Minor nit -- unrelated to this PR -- but it would be awesome to be able to see this written as something like:
```go
	c.AppContext = c.AppContext.With(app.RequestContextWithMaster)
```

Reply:
That is a fabulous idea that I had to adopt. Added via https://github.com/mattermost/mattermost/pull/24971/commits/68fcb562f5f4b4baed550d3305c1709a19242f6b.